research/gpu-exploration-app/ios/ExampleApp/Frameworks/MoproBindings.podspec

```
@@ -0,0 +1,11 @@
+Pod::Spec.new do |spec|
+  spec.name         = 'MoproBindings'
+  spec.version      = '0.1.0'
+  spec.summary      = 'MoproBindings XCFramework'
+  spec.homepage     = 'https://github.com/oskarth/mopro'
```

I just fixed the path in templates/mopro-example-app
```suggestion
  spec.homepage     = 'https://github.com/zkmopro/mopro'
```